bluemira/radiation_transport/flux_surfaces_maker.py

```diff
@@ -334,16 +342,25 @@ def _make_flux_surfaces(
     :
         The PartialOpenFluxSurface that passes through the point.
     """
-    coords = find_flux_surface_through_point(
+    coords_arr = find_flux_surface_through_point(
         equilibrium.x, equilibrium.z, equilibrium.psi(), x, z, equilibrium.psi(x, z)
     )
-    return OpenFluxSurface(Coordinates({"x": coords[0], "z": coords[1]})).split(
-        o_point, plane=yz_plane
-    )
+    coords = Coordinates({"x": coords_arr[0], "z": coords_arr[1]})
+    if dl is not None:
+        coords = coords.interpolate(dl=dl)
```

Comment:
this is the important line, I'm not currently preserving the points of the original fs but that can change